### DISCUSSION TITLE
Geotargeting limitations

### DISCUSSION THREAD

**Discussion by beshoo** [giuliopons/adadmin#95] [General]
Dear Programmer

I hope this email finds you well. I wanted to reach out to you regarding the ads banner management platform that you created. I have been using it for a while now and I find it very useful.

However, I have a concern about the geolocation function that targets only one country. As you know, my advertising campaigns target multiple countries, and I would like to restrict the display of banners to only those countries.

I am writing to request that you add multi-country targeting functionality to the platform. This would allow me to easily select the countries that I want to target and exclude the rest.

I understand that this may take some time and effort to implement, but I believe that it would greatly enhance the functionality of the platform and make it more useful for a wider range of users.

And we need farther status of banner geoloation if we chose all target to see which countries are view the banners and click on. 

Thank you for your attention to this matter. I look forward to hearing back from you and discussing this further.

Best regards,

**Comment by giuliopons** [comment]
Hi, sorry for my late reply.
When you say that you need multiple countries, do you also need multiple regions and city or only countries?
I have to think about the effort of this change because it's not easy.
Giulio

**Reply by beshoo** [reply]
Thanks for the reply.
Yes, only countries

**Comment by jefferose** [comment]
I came here to comment the same thing. For example, on affiliate program I use has a 3 programs for example. One for the UK specifically and one for Australia, and the rest use the US program. Right now, there's no way to target say "US and Canada" with a single banner. Creating 1 extra banner isn't huge work but when they have a bank of 20 offers and each one is now 4, it's a lot.

**Reply by giuliopons** [reply]
Hi @beshoo and @jefferose,
I understand your needs, I would be happy to proceed and add this feature but I need some funds, so I've configured the sponsorship program here on Github-Sponsors.

I've just added a 50$ level - **one time** - sponsorship.
If at least two supporters buy this tier I will add the feature.

It's the first time I try Github sponsorship service and It's the first time I try to split the new-feature cost to two different users, I hope you like this and that both of you can partecipate and share the cost!
https://github.com/sponsors/giuliopons

**Comment by beshoo** [comment]
If you like, i can programed it for free for you. Just share with me the
latest updated repository.

Regard

**Reply by giuliopons** [reply]
Although this is not free software, I like your help. So if you are able and you already have one of the latest version (tell me which version do you have, please) you can make the change and send me the updated files.
I will merge them on my server.
